mobile/test/objective-c/EnvoyTestServerInterface.mm

```diff
@@ -0,0 +1,33 @@
+#import "test/objective-c/EnvoyTestServerInterface.h"
+#import "test/common/integration/test_server_interface.h"
+
+@implementation TestServerInterface
```

Comment:
I'm not a huge obj-c user, but I'm surprised that we have a class named "Interface" with a concrete implementation. I would have expected an interface to be pure virtual.

Reply:
If this is no longer "pure virtual" I don't think we should call it Interface. Or did you change the name of this in a subsequent patch?

Reply:
It has been renamed yes